### PR TITLE
Added quickstarts/explain/index.md

### PR DESCRIPTION
**Pull request by joecheuk**

### CODE REVIEW

File: _docs/quickstarts/explain/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: IAM Explain
```
Comment (mirons-google) [inline _docs/quickstarts/explain/index.md]: s/IAM Explain/Explain/

File: _docs/quickstarts/explain/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+order: 202
```
Comment (mirons-google) [inline _docs/quickstarts/explain/index.md]: Ensure this order number works (view it rendered) and read the `README.md`

File: _docs/quickstarts/explain/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# {{ page.title }}
+
+This quickstart guide describes how to set up IAM Explain for Forseti Security.
+IAM Explain helps administrators, auditors, and users of Google Cloud to understand, 
+test, and develop Cloud Identity and Access Management (Cloud IAM) policies. 
+It can enumerate access by resource or member, answer why a principal has access 
+to a certain resource, or offer possible strategies for how to grant a specific 
+resource.
+
+
```
Comment (mirons-google) [inline _docs/quickstarts/explain/index.md]: nit: remove unnecessary newline

File: _docs/quickstarts/explain/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## Running IAM Explain
+
+Forseti Explain runs on a data model, so before you start using Explain, 
+you'll first select a model to use.
+
+You can use ‘–help’ to go through the hierarchy of commands and explore the 
+functionality. Only the basic examples are shown here.
+
+##### Selecting a data model
```
Comment (mirons-google) [inline _docs/quickstarts/explain/index.md]: We should incorporate more of the example from the testing guide.

File: _docs/quickstarts/explain/index.md (hunk 1 above)
Comment (joecheuk) [reply _docs/quickstarts/explain/index.md]: Updated.

File: _docs/quickstarts/explain/index.md (hunk 2 above)
Comment (joecheuk) [reply _docs/quickstarts/explain/index.md]: Updated.

File: _docs/quickstarts/explain/index.md (hunk 3 above)
Comment (joecheuk) [reply _docs/quickstarts/explain/index.md]: Updated.

File: _docs/quickstarts/explain/index.md (hunk 4 above)
Comment (joecheuk) [reply _docs/quickstarts/explain/index.md]: Updated.

**Comment by joecheuk** [comment]
Hi @Tabrarian , can you take a look at this PR when you have time? Thanks!

**Comment by joecheuk** [comment]
Thanks! @Tabrarian

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR & commit authors are googlers at the time of this PR was created.

**Pull request merged**